java/api/openapi.yaml

```diff
@@ -2511,10 +2511,18 @@ paths:
       - application/json
 components:
   schemas:
+    ProductType:
+      enum:
+      - INVOICE_FINANCING
+      - TERM_LOAN
+      - MCA
+      - LINE_OF_CREDIT
+      - INTEGRATED_MCA
+      type: string
     BusinessPrequalification:
       example:
         createdAt: 2022-06-01T03:57:26.115Z
-        product: TERM_LOAN
+        product: null
```

Comment:
Strange behavior on Java, Google a bit and do not see other people complaining about it somehow. Ask ChatGPT, and seems like it's known issue for the Java generator not taking example values into it's created examples 😂 Going to ignore it for now since we are not showing the example anywhere.


<img width="843" alt="Screenshot 2024-08-19 at 11 17 17" src="https://github.com/user-attachments/assets/f050ddba-e443-43f6-94bc-75a33e538ca3">